README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
Just wondered, because even the current hets-desktop doesn't have a runtime dependency to ncurses and the environment, where I usually build ubuntu packages does not have the libncurses-dev installed either ... Anyway, could be an indirect dep ...